components/image-controller/development/kustomization.yaml

```diff
@@ -2,8 +2,17 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - ../base
+- https://github.com/redhat-appstudio/image-controller/config/default?ref=8208ff372b880773b911062dc828f050e45c5312
+
+images:
+- name: quay.io/redhat-appstudio/image-controller
+  newName: quay.io/redhat-appstudio/image-controller
+  newTag: 8208ff372b880773b911062dc828f050e45c5312
+
+namespace: image-controller
 
 patches:
+  - path: ./manager_resources_patch.yaml
```

Comment:
Does the patch need to be copied to each environment or could it be only in `base`?

Reply:
since the patch is being applied to one of the resource coming from `https://github.com/redhat-appstudio/image-controller/config/default?ref=8208ff372b880773b911062dc828f050e45c5312` resource and we moved it to the individual overlays. 
When trying to apply the patch referring from base directory was getting error like `Error: trouble configuring builtin PatchTransformer with config: security; file 'manager_resources_patch.yaml' is not in or below`, 
I think there is a way to bypass this error, by using parameter 
`--load-restrictor string   if set to 'LoadRestrictionsNone', local kustomizations may load files from outside their root. This does, however, break the relocatability of the kustomization. (default "LoadRestrictionsRootOnly")`
when running `kustomize build`, but since it breaks `the relocatability of the kustomization` , I think it would be good to keep the patch inside each overlays

Reply:
Ack, makes sense